scripts/validate-headings.py

```diff
@@ -37,6 +37,9 @@ def validate_headers(path: Path) -> None:
                 admonition_is_note = line.startswith(":class: note")
                 assert admonition_is_note, f"{path=!s}: The admonition is not a note"
 
+            if line.startswith("# ") or line.startswith("## "):
```

Comment:
I think startswith accept tuples, but I'm not sure:
```suggestion
            if line.startswith(("# ", "## ")):
```